packages/components/package.json

```diff
@@ -136,6 +136,7 @@
       "./components/hds/application-state/footer.js": "./dist/_app_/components/hds/application-state/footer.js",
       "./components/hds/application-state/header.js": "./dist/_app_/components/hds/application-state/header.js",
       "./components/hds/application-state/index.js": "./dist/_app_/components/hds/application-state/index.js",
+      "./components/hds/application-state/media.js": "./dist/_app_/components/hds/application-state/media.js",
```

Comment:
Let's not forget to add the new component (and its subcomponents) also here:
https://github.com/hashicorp/design-system/blob/main/packages/components/src/components.ts

Reply:
We don't currently expose any of the subcomponents in that file, just the index component. Is this a change?

Reply:
I _think_ this is what we decided at the last HDS Engineering sync, no? https://docs.google.com/document/d/1twnk39hZ7p8oZYrld_FYsdiuUb6yn_weA46KnhF4nu4/edit#heading=h.uf036uudv34u

Reply:
Ah yes, good point. I'll update that in the morning to include those.